### PR TITLE
LT-1658: Expose inactive parties on case page

### PR DESCRIPTION
**Pull request by a-bek**
- Inactive parties table in case detail
- e2e tests

### CODE REVIEW

File: templates/case/includes/inactive-parties.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+						{% for flag in party.flags %}
+							{{ flag.name }}<br>
+						{% endfor %}
+					</td>
+				</tr>
+			{% endfor %}
+		</tbody>
+	</table>
+{% else %}
+	<p class="govuk-hint">
```
Comment (jfaracik) [inline templates/case/includes/inactive-parties.html]: Could you use the notice control?

Works like so:

`{% include "includes/notice.html" with text='CASE_ENTITIES_NONE' %}

File: ui_automation_tests/pages/application_page.py [hunk 2]
```diff
@@ -35,6 +35,7 @@ class ApplicationPage(BasePage):
     AUDIT_TRAIL_ITEM = ".app-activity__item"  # CSS
     APPLICATION_SUMMARY_BOARD = ".app-case-board"  # CSS
     TABLE_ENTITIES = "table-entities"  # ID
+    TABLE_INACTIVE_ENTITIES = "table-inactive-entities"  # ID
```
Comment (jfaracik) [inline ui_automation_tests/pages/application_page.py]: Suffix the constant with `_ID` and remove the comment :)

File: templates/case/applications/standard-licence-case.html [hunk 3]
```diff
@@ -186,6 +186,20 @@ <h2 class="govuk-accordion__section-heading">
 			</div>
 		</div>
 
+		<!-- Inactive Parties -->
+		<div class="govuk-accordion__section">
+			<div class="govuk-accordion__section-header">
+				<h2 class="govuk-accordion__section-heading">
+					<span class="govuk-accordion__section-button" id="accordion-default-heading-3">
+						{% lcs 'CASE_ENTITIES_INACTIVE' %}
```
Comment (andytaylorkainos) [inline templates/case/applications/standard-licence-case.html]: you'll need to change this string to use the class approach (`CASES.something.ENTITIES_INACTIVE`)

File: templates/case/includes/inactive-parties.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+		{% lcs 'CASE_ENTITIES_NONE' %}
```
Comment (andytaylorkainos) [inline templates/case/includes/inactive-parties.html]: same again for the class approach

File: templates/case/includes/inactive-parties.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+	</p>
+{% endif %}
```
Comment (andytaylorkainos) [inline templates/case/includes/inactive-parties.html]: newline

**Pull request merged**